### PR TITLE
<FunnelChart> - add new prop - displayValue

### PR DESCRIPTION
**Pull request by seifange**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6457

File: src/FunnelChart/FunnelLabel/FunnelLabel.js [hunk 1]
```diff
@@ -3,16 +3,14 @@ import PropTypes from 'prop-types';
...
 import Text from '../../Text';
 import { dataHooks } from '../constants';
-import { formatToCompactNumber } from '../../utils/numberFormatters';
-
 import { classes } from './FunnelLabel.st.css';
 
-export const FunnelLabel = ({ value, label }) => {
-  const compactNumber = value ? formatToCompactNumber(value, 1) : '-';
+export const FunnelLabel = ({ value, label, displayValue }) => {
+  const numberToPresent = displayValue ? displayValue : value ? value : '-';
```
Comment (EgoziE) [inline src/FunnelChart/FunnelLabel/FunnelLabel.js:9]: This condition is too complicated.
Lets add:
```
FunnelLabel.defaultProps = {
  value: '-',
}
```

**Review by EgoziE — CHANGES_REQUESTED** [review]
I can approve the visual tests, but maybe just add "displayValue" to them instead?

File: src/FunnelChart/FunnelLabel/FunnelLabel.js (hunk 1 above)
Comment (seifange) [reply src/FunnelChart/FunnelLabel/FunnelLabel.js:9]: sure np i'll fix!

**Review by EgoziE — APPROVED** [review]

**Pull request merged**